main.go

```diff
@@ -36,6 +40,13 @@ func main() {
 	bracket.SetStateFromSlice(players)
 	round := 1
```

Comment:
I think this should be set to `loadNum` if it's >= 0

Reply:
yeah agreed